libs/SalesforceSDK/res/values/sf__strings.xml

```diff
@@ -96,6 +96,10 @@
 	<string name="sf__biometric_opt_in_deny">Use Password</string>
 	<string name="sf__biometric_signout_user">Signing out %s.</string>
 
-	<!-- Dev support -->
-	<string name="sf__dev_support_title">Mobile SDK Dev Support</string>
+	<!-- Developer Support -->
+	<string name="sf__dev_support_title">Mobile SDK Developer Support</string>
```

Comment:
I placed these string resources here to match the location of the one existing "Developer Support" string.  Is this a match?  The content can change as we see fit.